tests/test_sycl_queue.py

```diff
@@ -148,31 +149,71 @@ def test_empty_like(device_x, device_y):
         pytest.param("zeros_like",
                      ['x0'],
                      {}),
+        pytest.param("tril",
+                     ['x0.reshape((2,2))'],
+                     {}),
+        pytest.param("triu",
+                     ['x0.reshape((2,2))'],
+                     {}),
         pytest.param("linspace",
                      ['x0', '4', '4'],
                      {}),
         pytest.param("linspace",
                      ['1', 'x0', '4'],
                      {})
     ])
-@pytest.mark.parametrize("device_x",
-                          valid_devices,
-                          ids=[device.filter_string for device in valid_devices])
-@pytest.mark.parametrize("device_y",
+@pytest.mark.parametrize("device",
                           valid_devices,
                           ids=[device.filter_string for device in valid_devices])
-def test_array_creation_like(func, args, kwargs, device_x, device_y):
-    x_orig = numpy.array([1, 2, 3])
+def test_array_creation_follow_device(func, args, kwargs, device):
+    x_orig = numpy.array([1, 2, 3, 4])
     numpy_args = [eval(val, {'x0' : x_orig}) for val in args]
     y_orig = getattr(numpy, func)(*numpy_args, **kwargs)
 
-    x = dpnp.array([1, 2, 3], device=device_x)
+    x = dpnp.array([1, 2, 3, 4], device=device)
     dpnp_args = [eval(val, {'x0' : x}) for val in args]
 
     y = getattr(dpnp, func)(*dpnp_args, **kwargs)
     numpy.testing.assert_allclose(y_orig, y)
     assert_sycl_queue_equal(y.sycl_queue, x.sycl_queue)
 
+
+@pytest.mark.parametrize(
+    "func, args, kwargs",
+    [
+        pytest.param("full_like",
+                     ['x0'],
+                     {'fill_value': 5}),
+        pytest.param("ones_like",
+                     ['x0'],
+                     {}),
+        pytest.param("zeros_like",
+                     ['x0'],
+                     {}),
+        pytest.param("linspace",
+                     ['x0', '4', '4'],
+                     {}),
+        pytest.param("linspace",
+                     ['1', 'x0', '4'],
+                     {})
+    ])
+@pytest.mark.parametrize("device_x",
+                          valid_devices,
+                          ids=[device.filter_string for device in valid_devices])
+@pytest.mark.parametrize("device_y",
+                          valid_devices,
+                          ids=[device.filter_string for device in valid_devices])
+def test_array_creation_cross_device(func, args, kwargs, device_x, device_y):
+    if func is 'linspace' and sys.platform.startswith('win'):
+        pytest.skip()
```

Comment:
```suggestion
        pytest.skip("CPU driver experiences an instability on Windows.")
```

Perhaps refer to an issue filed against `dpct`.